packages-exp/auth-exp/src/core/persistence/browser.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Persistence,
+  PersistenceType,
+  PersistenceValue,
+  Instantiator
+} from '.';
+
+const STORAGE_AVAILABLE_KEY_ = '__sak';
+
+class BrowserPersistence implements Persistence {
```

Comment:
should this live in platform_browser?

Reply:
I'm not sure. Apparently Cordova supports `localStorage` and `sessionStorage` but React native does not

Reply:
I see, if it's cross platform maybe it should stay here